packages/medusa/src/services/__mocks__/product.js

```diff
@@ -36,6 +36,8 @@ export const ProductServiceMock = {
     if (data.title === "Test Product") {
       return Promise.resolve(products.product1)
     }
+    if (data.title === "Test Product with variants")
+      return Promise.resolve(products.productWithOptions)
 
```

Comment:
Use braces for all control structures: https://github.com/airbnb/javascript#blocks